### PR TITLE
Enable GeometryCollection type in convex_hull_agg and ST_ConvexHull functions

### PR DESCRIPTION
**Pull request by yuyongyang800**
GeometryCollection type is supported in  both convex_hull_agg aggregate
function and ST_ConvexHull function.

### CODE REVIEW

File: presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/TestGeometryConvexHullGeoAggregation.java [hunk 1]
```diff
@@ -354,6 +352,53 @@
         };
     }
 
+    @DataProvider(name = "geometrycollection")
```
Comment (mbasmanova) [inline presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/TestGeometryConvexHullGeoAggregation.java]: s/geometrycollection/geometryCollection

**Comment by yuyongyang800** [comment]
Addressed the comments.

File: presto-docs/src/main/sphinx/functions/geospatial.rst [hunk 2]
```diff
@@ -325,7 +324,6 @@ Aggregations
 .. function:: convex_hull_agg(Geometry) -> Geometry
 
     Returns the minimum convex geometry that encloses all input geometries.
```
Comment (mbasmanova) [inline presto-docs/src/main/sphinx/functions/geospatial.rst]: Documentation change for `convex_hull_agg` needs to be in the same commit as the change itself.

File: presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java [hunk 3]
```diff
@@ -304,26 +304,13 @@ public static Slice stCentroid(@SqlType(GEOMETRY_TYPE_NAME) Slice input)
...
     {
         OGCGeometry geometry = deserialize(input);
-        validateType("ST_ConvexHull", geometry, EnumSet.of(POINT, MULTI_POINT, LINE_STRING, MULTI_LINE_STRING, POLYGON, MULTI_POLYGON));
         if (geometry.isEmpty()) {
             return input;
         }
         if (GeometryType.getForEsriGeometryType(geometry.geometryType()) == POINT) {
             return input;
         }
         OGCGeometry convexHull = geometry.convexHull();
```
Comment (mbasmanova) [inline presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java]: nit: inline this variable

**Pull request merged**